src/main/java/org/isf/utils/gui/PatientGuiConst.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Open Hospital (www.open-hospital.org)
+ * Copyright © 2006-2021 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
+ *
+ * Open Hospital is a free and open source software for healthcare data management.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * https://www.gnu.org/licenses/gpl-3.0-standalone.html
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.isf.utils.gui;
+
+public interface PatientGuiConst {
```

Comment:
I'm curious as to why these constants are in the CORE repo when they only appear to be used in GUI related code.....

Reply:
@mwithi asked me to move them in core.

Reply:
well, it is part of the issue, but I didn't request to do it in this way...
only the constant itself is supposed to be moved, not the whole class

moreover the issue is talking about two constants: `IMAGE_THUMBNAIL_MAX_WIDTH` and `MAXPROFPICFILESIZEBYTES`
I would move both into GeneralData (only in the code) so in the future we can have them controlled by the user... wdyt?